bats/fb-katello-content.bats

```diff
@@ -163,13 +163,10 @@ setup() {
   tSkipIfOlderThan41
   export_version=$(hammer --output csv --no-headers content-view version list --content-view="${CONTENT_VIEW}" --organization="${ORGANIZATION}"\
                --per-page=1 --fields="Version"  --order="version DESC")
-  export_repos=$(hammer --output csv --no-headers content-view version show --content-view="${CONTENT_VIEW}" --organization="${ORGANIZATION}" \
-    --version="${export_version}" --fields="Repositories/Name")
-  import_repos=$(hammer --output csv --no-headers content-view version show --content-view="${CONTENT_VIEW}" --organization="${IMPORT_ORG}" \
-    --version="${export_version}" --fields="Repositories/Name")
-
-  import_repos=$(echo $import_repos | tr "," "\n" | sort > import_repos)
-  export_repos=$(echo $export_repos | tr "," "\n" | sort > export_repos)
```

Comment:
This was always wrong to begin with IMHO. Assigning the variable with the same name as the filename is confusing.